lib/tasks/licenses.rake

```diff
@@ -9,28 +9,28 @@ namespace :licenses do  # rubocop:disable Metrics/BlockLength
     license_finder_report(report_path)
 
     warn 'License report generated.'
-    exit 0
   end
 
   desc 'Check license compliance of dependencies'
   task :compliance do
     warn 'Checking action items...'
     LicenseFinder::CLI::Main.new.action_items # Aborts if there are pending action items
 
+    # Skip report file checks if it is not under source control.
+    next unless File.exist?(report_path)
```

Comment:
> The comment is not accurate because it checks for the file presence

Comment is to state intention and this is the described intention is what is the purpose of the change. The implementation is usually easily understandable. But intention behind it can't be always captured by that. This is the case here. It is easy to see what is being done, but not easy to see why.

If you have a suggestion how to improve comment or implementation, that would be nice.

> Does Check action items check licenses compliance?

Yes.